### ISSUE TITLE
Duplicate Enemies Occupying Same Space

### ISSUE THREAD

**Issue by s3phir0th115** [DorkmasterFlek/smrpg_web_randomizer#82]
PLEASE DO NOT USE THIS FORM TO REQUEST FEATURES!  THIS IS FOR BUG REPORTS ONLY!
If you wish to discuss new features, please join our Discord server: http://discord.smrpgspeedruns.com

Console or Emulator: 
Console

If emulator state which emulator and version.  If console, state which flash cart firmware.
Console, but no flash cart. I'm using a custom SA-1 development cartridge with the ROM burned to a M27C322 EPROM.

Flags (paste flag string if available): 
Default Casual I believe

Seed number or hash (seed number or hash string from filename or download page): 
OR
Permalink (permalink URL from download page if you have it): 

SMRPG_US_8.2.7_open_2aa50555a5acc17c3340813c2af06d1b_1431774551.sfc

Description of problem: 
In at least 2 locations now, notably Kero Sewers and Rose Way, I've encountered enemies that seem to multiply themselves mid battle and occupy the same space. 2 of the same enemy will occupy the same space and when one is defeated, the other is able to be attacked and defeated as normal.

I've seen this with hobgoblins in Kero Sewers and arachnes on Rose Way.

I would expect them to be 2 separate enemies position wise. This behavior seems glitchy to me.

**Comment by DorkmasterFlek** [comment]
I haven't heard of this happening before.  Were you using Crowd Control by chance?

**Comment by s3phir0th115** [comment]
Nope, no Crowd Control.

After posting this, I did end up dumping the save from the cart and finishing the run on bsnes v115. The behavior is also exhibited with that.

[save.zip](https://github.com/DorkmasterFlek/smrpg_web_randomizer/files/5539432/save.zip)

I've attached a zip file with my save and a bsnes movie showing the behavior. I still have the ROM as well if it's needed to generate a patch or something.
